packages/gestalt-design-tokens/tokens/color/component.json

```diff
@@ -346,6 +346,18 @@
           }
         }
       },
+      "formfield": {
+        "primary": {
+          "value": "#fff",
```

Comment:
How come this isn't referring to our actual token? And it's hex values?

Reply:
Our lower level tokens don't cover this color.

![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/99115a17-6a91-4d9e-b5a7-59c3efb8ba84)


I'm prioritizing tokenizing vs creating lower level tokens which  should be design-driven

Reply:
Isnt' this the original token?

```
. whiteBg {
 background-color: var(--g-colorGray0);
}
```

Reply:
![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/7438f85a-389c-423e-83ed-4af1d477dbda)

Reply:
![Screenshot by Dropbox Capture](https://github.com/pinterest/gestalt/assets/10593890/6bcb7ce0-d68b-4f80-81a7-3b86aaa98eb9)

Reply:
@rlingineni Yes, it's that one.

Reply:
oh what, these aren't tokens?

Reply:
Nope.

Reply:
I called them "legacy colors" or color variables or old colors haha